### ISSUE TITLE
Can't load a cube.

### ISSUE THREAD

**Issue by marespiaut** [greenmoss/PyWavefront#75]
Hello,

I'm using PyWavefront 0.4.1 with Python 3.7.0 on Arch Linux.

I've tried loading a `cube.obj` downloaded at http://people.sc.fsu.edu/~jburkardt/data/obj/cube.obj but I get the following error message:

```
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/pywavefront/obj.py", line 379, in consume_faces
    self.next_line()
  File "/usr/lib/python3.7/site-packages/pywavefront/parser.py", line 106, in next_line
    self.line = next(self.lines)  # Will raise StopIteration when there are no more lines
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "tool2.py", line 71, in <module>
    App()
  File "tool2.py", line 28, in __init__
    obj = pywavefront.Wavefront('cube.obj')
  File "/usr/lib/python3.7/site-packages/pywavefront/wavefront.py", line 67, in __init__
    cache=cache)
  File "/usr/lib/python3.7/site-packages/pywavefront/obj.py", line 80, in __init__
    self.parse()
  File "/usr/lib/python3.7/site-packages/pywavefront/obj.py", line 90, in parse
    super(ObjParser, self).parse()
  File "/usr/lib/python3.7/site-packages/pywavefront/parser.py", line 136, in parse
    self.dispatcher.get(self.values[0], self.parse_fallback)()
  File "/usr/lib/python3.7/site-packages/pywavefront/obj.py", line 252, in parse_f
    self.material.vertices += list(self.consume_faces())
RuntimeError: generator raised StopIteration
```

**Comment by einarf** [comment]
Was fixed in #72 but it seems the issue was closed before a new version was released.

I'm also waiting for new release to get this fix.

**Comment by greenmoss** [comment]
I just released `0.4.2`; please let us know if it's working now.

**Comment by marespiaut** [comment]
Hello,

It seems to work fine now. Thank you for the fix!
